src/index.js

```diff
@@ -86,7 +86,11 @@ function handlebars(options) {
 
       for (var partial of scanner.partials) {
         // Register the partial dependencies as partials.
-        body += `import '${partial}${options.templateExtension}';\n`;
+        // do not import special partial @partial-block: http://handlebarsjs.com/partials.html#partial-block
+        if (!partial.startsWith('@')) {
+          // rollup-plugin-root-import require import from '/'
```

Comment:
Hm, could / should the developer take care of this requirement themselves by doing

```hbs
{{> '/views/_loading'}}

{{! The project-directory-relative path of the above template is '/src/client/js/views/_loading.html',
so the above import is made possible by registering `${__dirname}/src/client/js` as a
`rollup-plugin-root-import` `root` and a `rollup-plugin-handlebars-plus` `partialRoot`. }}
```

? That seems to be how we do things in our projects using `rollup-plugin-handlebars-plus` and `rollup-plugin-root-import`.